code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -190,6 +190,10 @@
 	if(nif)
 		compiled_vis |= nif.planes_visible()
 	//VOREStation Add End
+	//VOREStation Add - event hud
```

Comment:
Since an Add End is already before this line, might just remove it and leave the trailing end tag at the bottom. Minor nitpick, yes, but lesser number of lines as a whole. Can probably just leave this here at the suggested line.
```suggestion
	//event hud
```